features/standard/Autosave.feature

```diff
@@ -0,0 +1,43 @@
+@IbexaOSS @IbexaContent @IbexaExperience @IbexaCommerce @test1
+Feature: Content items creation
+  As an administrator
+  In order to manage content to my site
+  I want to check autosave feature during creation of Content Items
+
+  @javascript
+  Scenario: Content item is visible in draft dashboard after being autosaved
```

Comment:
I wouldn't do this one as admin. We can create another user here and lower the autosave value for him to something else than 60 seconds - for example 5 seconds.

Benefits:
1) We don't have a sleep for 60 seconds (this part of the test executes quicker)
2) We test that the autosave interval can be changed